object/token_cas.go

```diff
@@ -225,6 +235,9 @@ func GenerateCasToken(userId string, service string) (string, error) {
 		}
 
 		if value != "" {
+			if escapedValue, err := escapeXMLText(value); err == nil {
```

Comment:
What error will it generate? We usually return error if error found

Reply:
The error only occur in io.Write function, this escapeXMLText function is try to escape a original string to xml safe string, if it failed, it did nothing, and the response will use the original string like before. should I catch the error and break loop?

Reply:
@nanoguo if that error is unpected and critical, we should handle it, not ignore it

Reply:
ok